tests/ops/integration_tests/saas/test_hubspot_task.py

```diff
@@ -177,12 +218,14 @@ def test_saas_erasure_request_task(
         get_cached_data_for_erasures(privacy_request.id),
         db,
     )
+    config.execution.masking_strict = temp_masking
 
     # Masking request only issued to "contacts" and "subscription_preferences" endpoints
```

Comment:
Let's update this comment so we don't confuse anyone in the future.

Reply:
thanks, done